controlplane-rs/.dockerignore

```diff
@@ -0,0 +1,3 @@
+.vim/
+target/
+*.md
```

Comment:
I think longterm it would make sense to just do `controlplane` as opposed to `controlplane-rs`, right? We need to think about how this directory is going to look to someone new coming to the codebase who might not immediately understand the scope of our transition. A README in this directory explaining the transition and mentioning the Go code is the current controlplane for the time being would be helpful.